ocp_resources/utils.py

```diff
@@ -218,3 +219,19 @@ def remaining_time(self):
         Return the remaining part of timeout since the object was created.
         """
         return self.start_time + self.timeout - time.time()
+
+
+def get_resource_timeout_sampler(resource, status, timeout, sleep):
```

Comment:
`get_resource_timeout_sampler` rename to match the action
`get_resource_status_timeout_sampler`

Reply:
This function doesn't necessary gives you the status, since the status request is being done afterwards.
The Logger as well is a bit misleading since it expects you to get the status, while some 1 might get something else using this function.
It might be better to split the logger msg and the timeoutSampler creation code into 2 separate functions and use each of them on wait_for_status:
```
def get_resource_timeout_sampler
def print_status_log
```
What do you think?